plugins/CoreAdminHome/vue/src/JsTrackingCodeGenerator/JsTrackingCodeGenerator.vue

```diff
@@ -431,6 +433,24 @@ export default defineComponent({
         );
       }
 
+      if (!this.siteExcludedReferrers[idSite]) {
+        this.isLoading = true;
+
+        promises.push(
+          AjaxHelper.fetch({
+            module: 'API',
+            method: 'SitesManager.getExcludedReferrers',
+            idSite,
+            filter_limit: '-1',
+          }).then((data) => {
+            this.siteExcludedReferrers[idSite] = [];
+            (data || []).forEach((referrer: string) => {
```

Comment:
It looks like an object is being returned here so `.forEach` is failing and the generated tracker code isn't including `setExcludedReferrers`. Using `Object.values(...)` seems to fix it.
```suggestion
            Object.values(data || []).forEach((referrer: string) => {
```

Reply:
It should actually return an array. For me that worked without issues. Did you have any errors there?
Nevertheless I'll add the `Object.values`, as it wont hurt, even if it is already an array